In [37]:
import pandas as pd
import numpy as np
from scipy import stats

## Oue 1

In [2]:
# Pr(X<50)
1-stats.norm.cdf(50,loc = 45,scale = 8)

0.26598552904870054

## Oue 2

In [3]:
# Pr(X<38) and Pr(X<44)
pr_38 = stats.norm.cdf(38,loc = 38,scale = 6)
pr_44 = stats.norm.cdf(44,loc= 38,scale = 6)
#Pr(38<X<44)
print(pr_44,pr_44-pr_38)

0.8413447460685429 0.3413447460685429


In [4]:
# Pr(x>44)
1-pr_44

0.15865525393145707

In [5]:
pr_30 = stats.norm.cdf(30,loc = 38,scale = 6)
pr_30

0.09121121972586788

In [6]:
(400*9.12)/100

36.48

## Que 4

In [7]:
# Confidence interval for 99% confidence level
stats.norm.interval(0.99,loc= 100,scale= 20)

(48.48341392902199, 151.516586070978)

## Que 5

In [16]:
# Confidence interval for total profit
ci = stats.norm.interval(0.95,12,5)
ci

(2.2001800772997306, 21.79981992270027)

In [30]:
ci_in_rupee =(2.2001800772997306*45,21.79981992270027*45)

In [43]:
print(np.round(ci_in_rupee,4))

[ 99.0081 980.9919]


#### A. Range of the annual profit in Million Rupees is (99.0081, 980.9919)

In [59]:
stats.norm.ppf(.05,12,5)

3.7757318652426353

In [62]:
3.7757318652426353*45

169.90793393591858

#### B.The 5th percentile of profit is 169.908 Million Rupees

In [40]:
# Probability that the first division makes a loss is p1.
p1 = stats.norm.cdf(0,5,3)
p1

0.0477903522728147

In [41]:
# Probability that the second division makes a loss is p2.
p2 = stats.norm.cdf(0,7,4)
p2

0.040059156863817086

#### C. From p1 and p2, it is clear that first division has larger probability of making a loss.